src/bulk_insert.c

```diff
@@ -178,7 +179,7 @@ RedisModuleString** _Bulk_Insert_Insert_Nodes(RedisModuleCtx *ctx, RedisModuleSt
             LabelDesc l = labels[label_idx];
 
             // Label nodes.
-            Graph_LabelNodes(g, number_of_labeled_nodes, number_of_labeled_nodes + l.node_count -1,
+            Graph_LabelNodes(g, number_of_labeled_nodes + start_offset, number_of_labeled_nodes + start_offset + l.node_count,
```

Comment:
Repeated bulk inserts need to iterate only over new nodes.

Reply:
`number_of_labeled_nodes` is always 0 the first time this line is reached (set on line 164), and increases to match the number of nodes inserted with this label on line 195.

There is actually a bug on this line - the stopping condition should be `- 1`, which I'll add back momentarily.

Following the stopping condition fix, a sample execution of our stock `person` (11 node) and `country` (2 node) labels shows the following invocations:

`Graph_LabelNodes (g=[...], start_node_id=0, end_node_id=10, label=0, it=0x0)`
`Graph_LabelNodes (g=[...], start_node_id=11, end_node_id=12, label=1, it=0x0)`